packages/dart/npt_flutter/lib/features/profile_form/widgets/profile_device_at_sign_text_field.dart

```diff
@@ -30,14 +36,22 @@ class ProfileDeviceAtSignTextField extends StatelessWidget {
           },
           builder: (BuildContext context, String? state) {
             if (state == null) return gap0;
+            Future.microtask(() => controller.text = state);
             return SizedBox(
               width: Sizes.p300,
               height: Sizes.p80,
               child: TextFormField(
-                  initialValue: state,
+                  controller: controller,
                   autovalidateMode: AutovalidateMode.onUserInteraction,
                   validator: FormValidator.validateRequiredAtsignField,
                   onChanged: (value) {
+                    if (!value.startsWith('@')) {
+                      value = '@$value';
+                    }
+                    setState(() {
+                      value = value.trim();
+                    });
+
```

Comment:
What's the purpose of this change?

Reply:
Is the microtask for ensuring ui bound state remains in sync with the trimmed value?

Reply:
I don't believe I was the one that introduced this pattern, but I see what it's doing, and it looks fine to me.